### PR TITLE
Add ntfy to catalog

### PR DESCRIPTION
**Pull request by l-x**

### CODE REVIEW

**Comment by ericgaspar** [comment]
It would be nice to add ntfy to the [YunoHost-Apps](https://github.com/YunoHost-Apps) repo

**Comment by l-x** [comment]
> It would be nice to add ntfy to the [YunoHost-Apps](https://github.com/YunoHost-Apps) repo

"nice" as in "nice" or as in "required"? ^^

**Comment by ericgaspar** [comment]
ah... seems like Limezy already packaged the app...
@Limezy is your ntfy_ynh app working?

**Comment by Limezy** [comment]
My app is not working as I didn't have the time to build it yet. I'm fully ok to transfer everything obviously !

**Comment by Limezy** [comment]
@l-x are you member of the Yunohost apps org ? 
@ericgaspar how can I give rights on that repo to him ?

**Comment by l-x** [comment]
I am not a member (yet), sorry.

**Comment by ericgaspar** [comment]
@l-x I have sent you an invite

**Comment by ericgaspar** [comment]
I have added you as maintainer to https://github.com/YunoHost-Apps/ntfy_ynh

File: apps.json
Comment (ericgaspar) [inline apps.json:2945]:
```suggestion
        "url": "https://github.com/YunoHost-Apps/ntfy_ynh",
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: apps.json
Comment (l-x) [reply apps.json:2945]: Does that change make sense without changing state to `notworking`?

Comment (ericgaspar) [reply apps.json:2945]: we can merge this PR once the app is working

**Comment by l-x** [comment]
@Limezy Due to the fact there was no testing branch in https://github.com/YunoHost-Apps/ntfy_ynh I created that branch with my working changes.

**Review by ericgaspar — APPROVED** [review]

File: apps.json
Comment (ericgaspar) [inline apps.json:2945]:
```suggestion
        "url": "https://github.com/YunoHost-Apps/ntfy_ynh"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Pull request merged**
